mender-update/update_module/v3/update_module.hpp

```diff
@@ -48,11 +54,14 @@ enum class RebootAction { No, Automatic, Yes };
 
 using ExpectedRebootAction = expected::expected<RebootAction, Error>;
 
+using ExpectedWriterHandler = function<void(io::ExpectedAsyncWriterPtr)>;
```

Comment:
Q: Should this really be declared here?

Reply:
I can't think of a different use case for it at the moment. I'd leave it here for now, at least until we identify that it's needed elsewhere.